new branch without conflicts

robertgshaw2-redhat opened the pull request.

Commit:
new branch without conflicts

Commit:
updated with beth's changes

Beth-Kosis commented on src/content/user-guide/arm-alpha.mdx:
```diff
@@ -0,0 +1,213 @@
+- **No Performance Gains from Quantization**
+- **Limited Set of Models** (we have tested ResNet-50, YOLOv5, YOLACT, and BERT)
+- **AWS Graviton Only**
+
```
If the items above are not titles or proper names, only the first word should be capitalized. For example: Slower performance than DeepSparse x86

jeanniefinks reviewed (commented):
Also, there are a few things missing for me in terms of participation expectations and timeline. For example, here's a one-pager I did in 2020 for the engine which is meant to be an easy scannable page. It precedes a more formal playbook program content the user is to complete once we've accepted them into the product evaluation.  It could also be used for an email campaign. https://docs.google.com/document/d/1Q2cU8JVrW7NepwwGTX83LJUXY-I4O5bV/edit

Beth-Kosis commented on src/content/user-guide/arm-alpha.mdx:
```diff
@@ -0,0 +1,213 @@
+
+### System Requirements
+
+The General Release of DeepSparse for ARM will offer support for ARMv8+ devices, including server
```
General Release should be all lowercase.

Beth-Kosis commented on src/content/user-guide/arm-alpha.mdx:
```diff
@@ -0,0 +1,213 @@
+
+### Models
+
+The General Release of DeepSparse for ARM will offer support for all models in ONNX format. However, the 
```
General Release should be lowercase.

jeanniefinks replied:
```suggestion
The general release of DeepSparse for ARM will offer support for all models in ONNX format. However, the 
```

Beth-Kosis commented on src/content/user-guide/arm-alpha.mdx:
```diff
@@ -0,0 +1,213 @@
+### Models
+
+The General Release of DeepSparse for ARM will offer support for all models in ONNX format. However, the 
+ARM Alpha has been tested for accuracy on a subset a limited subset of models.
```
... on a limited subset of models.
I think "a subset" should be deleted.

jeanniefinks replied:
```suggestion
ARM Alpha has been tested for accuracy on a limited subset of models.
```

Beth-Kosis commented on src/content/user-guide/arm-alpha.mdx:
```diff
@@ -0,0 +1,213 @@
+
+The General Release of DeepSparse for ARM will offer support for all models in ONNX format. However, the 
+ARM Alpha has been tested for accuracy on a subset a limited subset of models.
+The following models have been tested for accuracy and performance with the DeepSparse ARM Alpha, 
```
Remove the comma at the end of this line.

jeanniefinks replied:
```suggestion
The following models have been tested for accuracy and performance with the DeepSparse ARM Alpha  
```

Beth-Kosis commented on src/content/user-guide/arm-alpha.mdx:
````diff
@@ -0,0 +1,213 @@
+pip install onnxruntime
+```
+
+At batch 64, ONNX runtime achieves 19 items/second running the standard BERT model.
````
ONNX Runtime
Because Runtime has an initial cap in previous instances, I'm assuming it's a proper name and should have an initial cap here as well.

jeanniefinks replied:
```suggestion
At batch 64, ONNX Runtime achieves 19 items/second running the standard BERT model.
```

Beth-Kosis commented on src/content/user-guide/arm-alpha.mdx:
````diff
@@ -0,0 +1,213 @@
+    --model_path "zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/12layer_pruned80_quant-none-vnni"
+```
+
+Making a request:
````
Here's an example of making a request:

jeanniefinks replied:
```suggestion
Here's an example of making a request:
```

jeanniefinks commented on src/content/user-guide/arm-alpha.mdx:
```diff
@@ -0,0 +1,213 @@
+
+DeepSparse has Alpha support for ARM platforms! This page explains how to trial it.
+
+[Sign up for early access to the general release of DeepSparse on ARM](https://neuralmagic.com/deepsparse-arm-waitlist)
```
```suggestion
[Sign up for early access to the general release of DeepSparse on ARM](https://neuralmagic.com/deepsparse-arm-waitlist).
```

jeanniefinks commented on src/content/user-guide/arm-alpha.mdx:
```diff
@@ -0,0 +1,213 @@
+
+## Want to Particpate in Our Trial?
+
+Neural Magic is offering a $50 Amazon Gift card for a 30 minute conversation to discuss: 
```
```suggestion
Neural Magic is offering a $50 Amazon gift card for a 30-minute conversation to discuss: 
```

jeanniefinks commented on src/content/user-guide/arm-alpha.mdx:
```diff
@@ -0,0 +1,213 @@
+
+Neural Magic is offering a $50 Amazon Gift card for a 30 minute conversation to discuss: 
+- Experience with the ARM Alpha
+- Requests for features for DeepSparse on ARM
```
```suggestion
- Feature requests for DeepSparse on ARM
```

jeanniefinks commented on src/content/user-guide/arm-alpha.mdx:
```diff
@@ -0,0 +1,213 @@
+- Requests for features for DeepSparse on ARM
+- Requests for ARM platform support (e.g. Raspberry Pi, Rockchip, Qualcomm)
+
+[Reach out to set up time to discuss](https://neuralmagic.com/contact/)
```
```suggestion
[Reach out to schedule a discussion now](https://neuralmagic.com/contact/)!
```

jeanniefinks commented on src/content/user-guide/arm-alpha.mdx:
```diff
@@ -0,0 +1,213 @@
+
+[Sign up for early access to the general release of DeepSparse on ARM](https://neuralmagic.com/deepsparse-arm-waitlist)
+
+## Want to Particpate in Our Trial?
```
```suggestion
## Want To Participate in Our Trial?
```

jeanniefinks commented on src/content/user-guide/arm-alpha.mdx:
```diff
@@ -0,0 +1,213 @@
+DeepSparse on ARM is currently in Alpha state. 
+
+It has the following limitations:
+- **Long Compile Times** (compiling BERT on `M6g.16xlarge` takes XXX minutes)
```
don't forget to fill in the xxx minutes (even if it's an approximate range if that was the intent here; it's hard to know if you are going to fill it in)

jeanniefinks commented on src/content/user-guide/arm-alpha.mdx:
```diff
@@ -0,0 +1,213 @@
+
+## Check Out Performance
+
+The following throughput statistics were generated using `deepsparse-nightly==xxx` on an `M6g.16xlarge` instance (64 cores):
```
which nightly was used here as it's not filled in?

Beth-Kosis commented on src/content/user-guide/arm-alpha.mdx:
```diff
@@ -0,0 +1,213 @@
+
+## Next Steps
+
+[Reach out to us to set up time to discuss your experience](https://neuralmagic.com/contact/)
```
Reach out to schedule a time to discuss your experience.

jeanniefinks replied:
```suggestion
[Schedule a time to discuss your experience](https://neuralmagic.com/contact/).
```

Beth-Kosis commented on src/content/user-guide/arm-alpha.mdx:
```diff
@@ -0,0 +1,213 @@
+
+[Reach out to us to set up time to discuss your experience](https://neuralmagic.com/contact/)
+
+[Sign up for early access to the general release of DeepSparse on ARM](https://neuralmagic.com/deepsparse-arm-waitlist)
```
Period at the end of the sentence.

jeanniefinks replied:
```suggestion
[Sign up for early access to the general release of DeepSparse on ARM](https://neuralmagic.com/deepsparse-arm-waitlist).
```

jeanniefinks commented on src/content/user-guide/arm-alpha.mdx:
```diff
@@ -0,0 +1,213 @@
+The following throughput statistics were generated using `deepsparse-nightly==xxx` on an `M6g.16xlarge` instance (64 cores):
+
+<p align="center">
+    <img width="90%" src="https://raw.githubusercontent.com/neuralmagic/docs/arm-alpha-new/src/images/arm-benchmarks.png"/>
```
```suggestion
    <img width="90%" src="https://raw.githubusercontent.com/neuralmagic/docs/arm-alpha-new/src/images/arm-benchmarks.png" alt = "ARM benchmark with DeepSparse on a M6g.16xlarge instance"/>
```

jeanniefinks commented on src/content/user-guide/arm-alpha.mdx:
```diff
@@ -0,0 +1,213 @@
+
+## Understand Alpha Status
+
+DeepSparse on ARM is currently in Alpha state. 
```
```suggestion
DeepSparse on ARM is currently in alpha state. It has the following limitations:
```

jeanniefinks commented on src/content/user-guide/arm-alpha.mdx:
```diff
@@ -0,0 +1,213 @@
+
+DeepSparse on ARM is currently in Alpha state. 
+
+It has the following limitations:
```
```suggestion
```

jeanniefinks commented on src/content/user-guide/arm-alpha.mdx:
```diff
@@ -0,0 +1,213 @@
+
+# DeepSparse ARM Alpha
+
+DeepSparse has Alpha support for ARM platforms! This page explains how to trial it.
```
```suggestion
DeepSparse has alpha support for ARM platforms! This page explains how to trial it.
```

jeanniefinks reviewed (changes requested):
Made suggested edits.

jeanniefinks committed:
Update arm-alpha.mdx

Minor grammar edits; removed "Amazon" gift card vendor in case selected users don't use Amazon in their geographi carea.

robertgshaw2-redhat committed:
Update arm-alpha.mdx

robertgshaw2-redhat committed:
Update arm-alpha.mdx

robertgshaw2-redhat committed:
Update arm-alpha.mdx

robertgshaw2-redhat committed:
Update arm-alpha.mdx

Commit:
updated graphic

robertgshaw2-redhat committed:
Update arm-alpha.mdx

Commit:
updated image

robertgshaw2-redhat committed:
Update arm-alpha.mdx

Commit:
updated image

Commit:
Merge branch 'arm-alpha-new' of github.com:neuralmagic/docs into arm-alpha-new

Commit:
removed

Commit:
added back

robertgshaw2-redhat committed:
Update arm-alpha.mdx

robertgshaw2-redhat committed:
Update image-classification.mdx

robertgshaw2-redhat committed:
Update natural-language-processing.mdx

robertgshaw2-redhat committed:
Update object-detection.mdx
